mod_reforged/hooks/skills/perks/perk_student.nut

```diff
@@ -0,0 +1,3 @@
+::mods_hookExactClass("skills/perks/perk_student", function (o) {
```

Comment:
You should hook `create` and set `this.m.IsRefundable` to false during the create.